media/nls/src/c_037.txt

```diff
@@ -1,616 +1,621 @@
-﻿CODEPAGE 37
+CODEPAGE 037            ;IBM EBCDIC U.S./Canada
```

Comment:
Quite surprised they have a leading zero; it looks like octal.

Reply:
Yes, it actually bit me while testing this. This handled with stol called with base=10